packages/commonwealth/client/scripts/views/components/component_kit/cw_cover_image_uploader.tsx

```diff
@@ -482,7 +481,6 @@ export const CWCoverImageUploader = ({
             name="image-behaviour"
             onChange={(e) => {
               setImageBehavior(e.target.value);
-              uploadCompleteCallback(imageURL, e.target.value);
```

Comment:
do we need to remove this?

Reply:
@mzparacha Just added these 2 back in and made them optional. Thanks for the feedback.